### PR TITLE
creates nohup files for elasticsearch, logstash and kibana in the log…

### PR DESCRIPTION
**Pull request by alanc94**
Separate nohup files created for Elasticsearch, Logstash and Kibana and put into their respective folder in the "logs" folder. 
vm.max_map_count is no longer needed in startES.sh, no sudo access reqired running elasticsearch.

### CODE REVIEW

File: bin/logstashScripts/logstash.conf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 input {
     file {
-        path => "{ELK_SRC_HOME}**/*.java"
+        path => "/home/alan/gitfolder/jersey/**/*.java"
```
Comment (bazzani) [inline bin/logstashScripts/logstash.conf]: I think you might want to revert this change

**Review by bazzani — CHANGES_REQUESTED** [review]

**Review by balajisridhar — APPROVED** [review]

**Review by bazzani — APPROVED** [review]
